### ISSUE TITLE
Activity not tracking. Showing `Unknown`

### ISSUE THREAD

**Issue by hkchakladar** [ActivityWatch/aw-android#99]
Showing `Unknown`. See screenshot.
![Screenshot_20231006-004720_ActivityWatch](https://github.com/ActivityWatch/aw-android/assets/7250906/e8a83b7d-e087-4055-bbef-cf907013e54f)

**Comment by ErikBjare** [comment]
I discovered this too, for some reason it can't get the app names anymore.

I did some investigating trying to hunt down the core issue, but I couldn't figure it out.

Here's the code:

https://github.com/ActivityWatch/aw-android/blob/0f095203f25b7036d12ad49720a2dd3dd5afab37/mobile/src/main/java/net/activitywatch/android/models/Event.kt#L13-L20

Made a new release which at least includes the package name, such that it should at least say, for example, `Unknown (com.twitter.android)`.
